Management/src/ActuatorWeb/Pages/Weather.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@page
+@model WeatherModel
+@{
+    ViewData["Title"] = "Weather Forecast";
+}
+<h1>@ViewData["Title"]</h1>
+@if (Model.RequestException != null)
```

Comment:
Getting weather forecasts still has many problems. Instead of pointing them all out, I've fixed them in the fix-weather branch (last commit).